Meta/Lagom/CMakeLists.txt

```diff
@@ -337,6 +337,10 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
     # NetBSD has its shm_open and shm_unlink functions in librt so we need to link that
     target_link_libraries(LibCore PRIVATE librt.so)
 endif()
+if (${CMAKE_SYSTEM_NAME} MATCHES "SunOS")
+    # Solaris has socket and networking related functions in two extra libraries
+    target_link_libraries(LibCore PRIVATE libnsl.so libsocket.so)
```

Comment:
I think typical cmake etiquette here would be 

```
target_link_lbiraries(LibCore PRIVATE nsl socket)
```
rather than explicitly linking the shared libraries.

Reply:
Ah I see this is following what we did for NetBSD above. hm. I'll probably submit a patch soon to make that section more idiomatic then.